Sources/Layout/MessageSizeCalculator.swift

```diff
@@ -107,7 +107,7 @@ open class MessageSizeCalculator: CellSizeCalculator {
 
     // MARK: - Avatar
 
-    internal func avatarPosition(for message: MessageType) -> AvatarPosition {
+    open func avatarPosition(for message: MessageType) -> AvatarPosition {
```

Comment:
Not sure if I wanted these `open` 🤔 cc @zhongwuzw

Reply:
`internal` would be fine, we use `incoming*` and `outgoing*` properties to let layout static, if we make it `open`, it's back to the original dynamic way.

Reply:
Exactly what I was thinking 👍

Reply:
Ah, I see. I was thinking someone might want to alter the default behavior - seemed arbitrary not to allow that.